pages/steve.js

```diff
@@ -0,0 +1,367 @@
+import { Box, Link, Grid, Image, Container, Button, Heading, Text, Input, Checkbox } from 'theme-ui'
+import Head from 'next/head'
+import Meta from '@hackclub/meta'
+import ForceTheme from '../components/force-theme'
+import Footer from '../components/footer'
+import Nav from '../components/nav'
+import Tilt from '../components/tilt'
+import Ticker from 'react-ticker'
+import { useState, useEffect } from 'react'
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const StevePage = () => {
+
+  const [startDate, setStartDate] = useState(null);
+  const [endDate, setEndDate] = useState(null);
+
+  const [disabledDates, setDisabledDates] = useState([]);
+
+  useEffect(() => {
+    // Fetch the disabled dates from the API endpoint when the component mounts
+    const fetchDisabledDates = async () => {
+      try {
+        const response = await fetch('/api/steve');
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+
+        const dateStrings = await response.json();
+        const dateObjects = dateStrings.map(dateStr => {
+            const date = new Date(Date.UTC(dateStr.substring(0, 4), dateStr.substring(5, 7) - 1, dateStr.substring(8, 10)));
+            date.setDate(date.getDate() + 1); // Add one day to the date
+            return date;
+        });
+        setDisabledDates(dateObjects);
+      } catch (error) {
+        console.error('Failed fetching disabled dates:', error);
+      }
+    };
+
+    fetchDisabledDates();
+}, []);
+
+
```

Comment:
This whitespace is probably unnecessary, but it actually bring me to wonder if we do (or could do) any linting that might flag things like this...